TriggertrapSLR/TTViewController.swift

```diff
@@ -164,14 +165,15 @@ class TTViewController: SplitLayoutViewController, DispatchableLifecycle, Sequen
         let delay = 0.3
         
         DispatchQueue.main.asyncAfter(deadline: DispatchTime.now() + Double(Int64(delay * Double(NSEC_PER_SEC))) / Double(NSEC_PER_SEC), execute: {
-            
-            // Only show the notification again if it is not currently presented
-            if !(self.notification?.isAnimating)! {
-                self.notification?.backgroundColor = UIColor.triggertrap_primaryColor()
-                self.notification?.titleColor = UIColor.triggertrap_fillColor()
-                self.notification?.show()
-                self.notification?.duration = 2.0
-            }
+            self.notification?.backgroundColor = UIColor.triggertrap_primaryColor()
+            self.notification?.titleColor = UIColor.triggertrap_fillColor()
+            self.notification?.duration = 2.0
+            self.notification?.show()
+            //this needs to be reinstantiated each time to avoid glitchy behavior (sadly)
+            self.notification = MPGNotification(title: NSLocalizedString("Low Volume", comment: "Low Volume"),
```

Comment:
@alex-taffe can you explain a bit more the reasoning for this? What exactly is the "glitchy behaviour" and how does this resolve it?

Also, why have we removed the check for `isAnimating`, will that not cause multiple notifications to stack on-top of each other?